resources/default_pipeline_environment.yml

```diff
@@ -252,6 +252,9 @@ steps:
     newmanRunCommand: "run '${config.newmanCollection}' --environment '${config.newmanEnvironment}' --globals '${config.newmanGlobals}' --reporters junit,html --reporter-junit-export 'target/newman/TEST-${collectionDisplayName}.xml' --reporter-html-export 'target/newman/TEST-${collectionDisplayName}.html'"
     stashContent:
       - 'tests'
+  npmExecute:
+    dockerImage: 'node:8-stretch'
+    npmCommand: 'install'
```

Comment:
I would not like to have a default command configured that is something with should be defined (only) over the parameter.

Reply:
done with https://github.com/SAP/jenkins-library/pull/491/commits/0b71f3d9a2ea0ba909381403479a1211bc37cd06